src/Static/Db.elm

```diff
@@ -39,60 +26,39 @@ type alias Db =
     }
 
 
-db : Result String Db
-db =
-    Result.map5 Db impactDefinitions textileDb foodDb countries distances
+db : StaticJson.Processes -> Result String Db
+db procs =
+    StaticJson.db procs
+        |> Result.andThen
+            (\{ foodDb, textileDb } ->
+                Result.map3
+                    (\okImpactDefinitions okCountries okDistances ->
+                        Db okImpactDefinitions textileDb foodDb okCountries okDistances
+                    )
```

Comment:
Nit: this can be `(\okImpactDefinitions -> Db okImpactDefinitions textileDb foodDb)`